packages/product-catalogue/src/components/ProductForm/index.tsx

```diff
@@ -21,7 +21,7 @@ export interface ProductFormFields {
   appropriateUsage: string;
```

Comment:
Do we want the strings in this component to come from some constants/lang module?  What will we do when we need to make this translatable?